src/psyclone/tests/domain/gocean/transformations/globalstoargs_test.py

```diff
@@ -336,11 +336,20 @@ def test_globalstoarguments_noglobals():
                            api=API)
     psy = PSyFactory(API).create(invoke_info)
     invoke = psy.invokes.invoke_list[0]
-    kernel = invoke.schedule.coded_kernels()[0]
-    trans = KernelGlobalsToArguments()
     before_code = str(psy.gen)
+
+    # Parse the same file but this time apply the KernelGlobalsToArguments
+    _, invoke_info = parse(os.path.join(BASEPATH, "gocean1p0",
```

Comment:
Do we need to repeat the parsing - can we just use the old `invoke_info`?

Reply:
I found some errors reusing it, so for the time being I added relevant TODOs that may fix them.

Reply:
Removed the TODO, using fortran_writer this is fixed and we don't need to repeat the parsing.